tests/nbsp.txt

```diff
@@ -1 +1 @@
-hello world & people < > &NBSP;
\ No newline at end of file
+hello   world & people < > &NBSP;
```

Comment:
No, actually I've corrected it.
<img width="262" alt="screen shot 2017-08-09 at 1 24 28 am" src="https://user-images.githubusercontent.com/5491415/29112095-930c9be0-7ca1-11e7-979a-32b43d20bb11.png">


Rendered HTML screenshot (Firefox):